src/linear_algebra/basis.lean

```diff
@@ -880,7 +880,7 @@ begin
     intro hi,
     simp [hi] },
   { refine top_unique (λ _ _, _),
-    simp [submodule.mem_span_singleton] }
+    simp only [mem_span_singleton, range_const, mul_one, exists_eq, smul_eq_mul] }
```

Comment:
`simp [submodule.mem_span_singleton]` was failing, maybe due to the new import in `linear_algebra.basic`?